tools/Makefile.am

```diff
@@ -4,10 +4,10 @@ noinst_HEADERS = tools_common.h
 AM_CPPFLAGS = -I../include
 
 dhtnode_SOURCES = dhtnode.cpp
-dhtnode_LDFLAGS = -lopendht -lreadline -L../src/.libs @GnuTLS_LIBS@
+dhtnode_LDFLAGS = -lopendht -lreadline -L@top_builddir@/src/.libs -L@top_builddir@/argon2/src/.libs @GnuTLS_LIBS@
```

Comment:
local argon2 path included even when system library is used. might use @argon2_LIBS@ ?